youtube_dl/extractor/mtv.py

```diff
@@ -170,12 +178,7 @@ def _get_video_info(self, itemdoc):
     def _get_videos_info(self, uri):
         video_id = self._id_from_uri(uri)
         feed_url = self._get_feed_url(uri)
-        data = compat_urllib_parse.urlencode({'uri': uri})
-        info_url = feed_url + '?'
-        if self._LANG:
-            info_url += 'lang=%s&' % self._LANG
-        info_url += data
-        return self._get_videos_info_from_url(info_url, video_id)
+        return self._get_videos_info_from_url(feed_url, video_id)
```

Comment:
Thanks, I've refactored it a bit and `MTVServicesEmbeddedIE` test should now pass again.